scripts/scil_run_nlmeans.py

```diff
@@ -96,7 +88,7 @@ def main():
     if args.logfile is not None:
         log.addHandler(logging.FileHandler(args.logfile, mode='w'))
 
-    vol = nb.load(args.input)
+    vol = nb.load(args.in_image)
     data = vol.get_fdata(dtype=np.float32)
```

Comment:
Isn't that the default?

Reply:
Yes but to make it clear in all scripts we ask to be explicit about this.